src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder saveDataToStringFromExistFile = new StringBuilder();
+
+        try {
+            BufferedReader readerFromExistFile = new BufferedReader(new FileReader(fromFileName));
+            String value = readerFromExistFile.readLine();
+
+            while (value != null) {
+                saveDataToStringFromExistFile.append(value).append(" ");
+                value = readerFromExistFile.readLine();
+            }
+
+        } catch (IOException e) {
+            throw new RuntimeException("File is not found ",e);
+        }
+
+        createReport(calculateValue(saveDataToStringFromExistFile), toFileName);
+
+    }
+
+    public int[] calculateValue(StringBuilder stringBuilderValues) {
+
+        String[] splitSaveDatas = stringBuilderValues.toString().split("\\s+");
+        int supply = 0;
+        int buy = 0;
+        int result;
+
+        for (String splitSaveData : splitSaveDatas) {
+            String[] splitValue = splitSaveData.split(",");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
            String[] splitValue = splitSaveData.split(DELIMITER);
```